tutorials/networking/high_level_multiplayer.rst

```diff
@@ -371,9 +372,151 @@ have loaded the game scene.
         players.clear()
         server_disconnected.emit()
 
+ .. code-tab:: csharp C#
+    using Godot;
+    using Godot.Collections;
+
+    public partial class Lobby : Node
+    {
+        // These signals can be connected to by a UI lobby scene or the game scene.
+        [Signal]
+        public delegate void PlayerConnectedEventHandler(int peerId, Dictionary<string, string> playerInfo);
+
+        [Signal]
+        public delegate void PlayerDisconnectedEventHandler(int peerId);
+
+        [Signal]
+        public delegate void ServerDisconnectedEventHandler();
+
+        private const int _port = 7000;
+        private const string _defaultServerIp = "127.0.0.1";
+        private const int _maxConnections = 20;
```

Comment:
We use PascalCase for constants, and IP should be all uppercase because it's a 2-letter acronym. Also, missing a comment about localhost.

```suggestion
        private const int Port = 7000;
        private const string DefaultServerIP = "127.0.0.1"; // IPv4 localhost
        private const int MaxConnections = 20;
```